lms/djangoapps/grades/models.py

```diff
@@ -679,6 +680,9 @@ def _cache_key(cls, course_id):
     def _emit_grade_calculated_event(grade):
         events.course_grade_calculated(grade)
 
+    @staticmethod
+    def _emit_persistent_grade_summary_changed_event(course_id, user_id, grade):
```

Comment:
Could you add a docstring to know what this function does?

Reply:
Sorry, I forget to add this docstring, but I have already fixed it.